spond/club.py

```diff
@@ -0,0 +1,59 @@
+import aiohttp
+
+from .base import SpondBase
+
+
+class SpondClub(SpondBase):
+    def __init__(self, username, password):
+        super().__init__(username, password, "https://api.spond.com/club/v1/")
+        self.transactions = None
+
+    @SpondBase.require_authentication
+    async def get_transactions(
+        self, club_id: str, skip: int = None, max_items: int = 100
```

Comment:
`skip` is None by default, so type hint is wrong - should be `Optional [int]` or `int | None` - former matches rest of codebase style

Reply:
done!